app/models/follow_up.rb

```diff
@@ -70,6 +70,7 @@ def human_status
   end
 
   def current_pending_rdv
-    participations.select(&:pending?).min_by(&:starts_at)
+    # if rdv is the same day, it is not in the future therefore not considered "pending", but the follow-up is still
+    participations.select(&:pending?).min_by(&:starts_at) || participations.select(&:unknown?).min_by(&:starts_at)
```

Comment:
```suggestion
    participations.select(&:pending?).min_by(&:starts_at) || participations.select(&:unknown?).max_by(&:starts_at)
```
parce que si on a des rdvs dans le passé où le statut n'a pas été renseigné ça affichera une mauvaise date ici.
Ç'aurait été top d'avoir un petit test pour le bugfix